pysqldb3/query.py

```diff
@@ -537,15 +537,15 @@ def print_query(query_string):
             print(_)
 
     @staticmethod
-    def query_to_gpkg(dbo, query, path=None, gpkg_name=None, cmd=None, gpkg_tbl=None, gdal_data_loc=GDAL_DATA_LOC, print_cmd=False, srid=2263):
+    def query_to_gpkg(dbo, query, path, gpkg_name, gpkg_tbl, cmd=None, gdal_data_loc=GDAL_DATA_LOC, print_cmd=False, srid=2263):
```

Comment:
why is this static?

Reply:
Because `query_to_shp` was static so I thought it would remain consistent

Reply:
odd i cant really imagine why it should be since it requires the dbo